src/style.css

```diff
@@ -0,0 +1,79 @@
+@font-face {
+  font-family: Avenir-Book;
+  src: url(fonts\Avenir-Book.ttf);
+}
+@font-face {
+  font-family: Avenir-Heavy;
+  src: url(fonts/Avenir-Heavy.ttf);
+}
+
+body {
+  display: block;
+  font-weight: 300;
+}
+
+.form {
+  margin-top: 20px;
+  border: 1px solid #e1e7ed;
+  border-radius: 4px;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+}
+
+.form-1 {
+  height: 68px;
+}
+
+.form-2 {
+  height: 40px;
+}
+
+.big__img {
+  margin-right: 17px;
+  margin-left: 25px;
+  margin-top: 24px;
+}
+
+.big__search {
+  border: none;
+  display: block;
+  position: absolute;
+  top: 43px;
+  left: 70px;
+  font-family: Avenir-Book, Arial, Helvetica, sans-serif;
+  font-size: 16px;
+  line-height: 22px;
+}
+
+.small__img {
+  margin-left: 12px;
+  margin-right: 9px;
+  margin-top: 14px;
+  width: 11px;
+  height: 11px;
+}
+
+.small__search {
+  border: none;
+  display: block;
+  position: absolute;
+  left: 41px;
+  top: 120px;
+  font-family: Avenir-Book, Arial, Helvetica, sans-serif;
+  font-size: 14px;
+  line-height: 19px;
+}
+
+.small__search::placeholder,
+.big__search::placeholder {
+  color: #3d4e61;
+}
+
+.small__search:hover,
+.big__search:hover {
+  border: 1px solid #3d4e61;
+}
+
+.small__search:focus,
+.big__search:focus {
+  font-family: Avenir-Heavy, Arial, Helvetica, sans-serif;
+}
```

Comment:
Again, you think in the right direction, but let's make it even more simple.
You'll get to know more about this issue when you'll learn BEM methodology next week.

Let's select some base class for search input.
You already have `search` class that you don't style at all, by the way.
Our big input will be some base option.
Into this class we'll add all the styles we need for big input (incl. :hover, ::placeholder etc.)
We'll apply this class to big input in HTML (`<input class="search" ... `)

Then we have option for small input.
We'll apply two classes here - base class and class with modifier: (`<input class="search search--small" ...`)
And into `search--small` class in CSS we'll add all styles that small input should have, but only those, that differ from big search. So we won't have to write `position: absolute;` or `display: block;` for the second time.